frontend/src/app/modules/workspace/video/video-description/video-description.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-video-description',
+  templateUrl: './video-description.component.html',
+  styleUrls: ['./video-description.component.scss'],
+})
+export class VideoDescriptionComponent {
+  @Input() public Username?: string;
+  @Input() public DateOfComment?: string;
```

Comment:
```suggestion
  @Input() public username?: string;
  @Input() public dateOfComment?: string;
```

?
But maybe I'm wrong. Maybe there is some specific idea behind naming it this way